software/firmware/setup.py

```diff
@@ -1,10 +1,11 @@
-import sys
-
 from setuptools import setup
 
+__version__ = ""
+exec(open("./version.py").read())
+
 setup(
     name="micropython-europi",
-    version="0.6.1",
+    version=__version__,
```

Comment:
[required] This will require an update to the release process in `./github/workflows/README.md`.